services/tenant-ui/frontend/src/components/layout/Sidebar.vue

```diff
@@ -2,6 +2,7 @@
   <div class="traction-sidebar">
     <!-- <h1 v-if="tenant" class="sidebar-app-title">{{ tenant.name }}</h1> -->
     <h1 class="sidebar-app-title">Tenant UI</h1>
+    <h1 class="sidebar-app-title small">T</h1>
```

Comment:
This is a placeholder so it might not be a T, will shortly be putting in the /tenant call to load the name for this header.
Probably just leave for now and I can put the logic in to take the first name as part of a computed property when that happens. Unless you know some CSS that would just take the first letter of whatever content or something.

Reply:
Hmmm.... You can specify content in css depending on width. But I don't there there's a way to say just use first letter.